lib/iris/tests/integration/plot/test_colorbar.py

```diff
@@ -46,49 +43,37 @@ def test_common_draw_functions(self):
         for draw_function in self.draw_functions:
             mappable = draw_function(self.cube)
             cbar = plt.colorbar()
-            self.assertIs(
-                cbar.mappable,
-                mappable,
-                msg="Problem with draw function iris.plot.{}".format(
-                    draw_function.__name__
-                ),
-            )
+            assert (
```

Comment:
I tend to not have bracket surrounding the assert, I find it less readable. Persoanl preference though.